lib/public_key/test/public_key_SUITE.erl

```diff
@@ -1369,6 +1369,25 @@ cacerts_load() ->
 cacerts_load(Config) ->
```

Comment:
due past events, I would add some file links verification to make sure they work as expected